src/arviz_plots/backend/none/__init__.py

```diff
@@ -398,7 +398,7 @@ def ciliney(
     if not ALLOW_KWARGS and artist_kws:
         raise ValueError("artist_kws not empty")
     artist_element = {
-        "function": "line",
+        "function": "ciliney",
```

Comment:
quite off scope but I did realize here. This funcion ends up calling `plot` which is the same artist that is called by `line`, the difference is it calls `plot([x, x], [y_bottom, y_top]...)` but this [x, x] transform/cleanup should be done at the `visuals` level to keep the backend level as minimal as possible.

Extra note: if using [`vlines`](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.vlines.html) here for whatever reason (performance, better fit for non matplotlib backends...) then it would make sense to have it as its own backend function